src/index.js

```diff
@@ -56,6 +56,15 @@ const MicroModal = (() => {
     }
 
     showModal () {
+      const isOpenClassName = 'is-open'
+      if (this.modal.classList.contains(isOpenClassName)) {
+        if (this.config.debugMode) console.warn('Modal already open')
+        return
+      }
+      openModals.push(this)
+      let parentDOMNode = this.modal.parentNode
```

Comment:
In order to stack the modals correctly, they need to be added to the DOM in the order they are to be stacked.  So this code removes the modal from the DOM and then adds it back to the DOM.